src/mainboard/pcengines/apu3/mainboard.c

```diff
@@ -50,6 +50,37 @@
 #include <cpuRegisters.h>
 #include <build.h>
 #include "bios_knobs.h"
+#include <spi-generic.h>
```

Comment:
@miczyg1 this bad approach to copy that code everywhere - as future improvement please create a separate code module which you will include for each platform. Please submit JIRA ticket for that.